### DISCUSSION TITLE
[Feature] 应用商店支持nginx_proxy_manager汉化版本

### DISCUSSION THREAD

**Discussion by cyrinkwork** [1Panel-dev/1Panel#7183] [Ideas]
### 1Panel 版本

v1.10.21-lts

### 请描述您的需求或者改进建议

应用商店 nginx_proxy_manager 有一个汉化版本，源码地址 https://github.com/xiaoxinpro/nginx-proxy-manager-zh ，是否可以更新为汉化版本

**Comment by zhengkunwang223** [comment]
目前不会 你可以通过修改 docker-compose 文件中镜像部分来使用这个版本
